genesis/engine/entities/rigid_entity/rigid_entity.py

```diff
@@ -351,20 +351,14 @@ def _load_MJCF(self, morph, surface):
 
         for i_e in range(mj.neq):
             e_info = mju.parse_equality(mj, i_e, morph.scale, ordered_links_idx)
-            if e_info["type"] == gs.EQUALITY_TYPE.CONNECT:  # only this type is supported right now
```

Comment:
Missing error for `WELD` equality type

Reply:
yeah, I want to add WELD in the next PR.

Reply:
Yes, so the check should be kept for now 😄

Reply:
```
            # only those two types of equality are supported
            if e_info["type"] == gs.EQUALITY_TYPE.CONNECT or e_info["type"] == gs.EQUALITY_TYPE.JOINT: 
```